scripts/versioning/check-contracts.ts

```diff
@@ -0,0 +1,13 @@
+import {Box, Version} from '../../typechain'
```

Comment:
question: the release yaml invokes this script. Is there a GitHub issue open to add versioning to the existing contracts (with adding / replacing / expanding this check version logic)?

Reply:
added #184